src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -108,6 +108,12 @@ export class MeetingManager implements AudioVideoObserver {
 
   private eventDidReceiveRef: EventObserver;
 
+  private _deviceLabels: DeviceLabels | DeviceLabelTrigger;
+  
+  public get deviceLabels() : DeviceLabels | DeviceLabelTrigger {
```

Comment:
nit: The component library does not appear to use the getter, except `Versioning.ts`. How about making `getDeviceLabels()`?

Reply:
Sure, updated to use `getDeviceLabels` over `getter`.